GSoC-interest.md

```diff
@@ -21,6 +21,9 @@ GSoC Mentors
 | Name | Email | Idea | Micro-Task Repository | Project Proposal |
 | --- | --- | --- | --- | --- |
 | Georg Link | glink at unomaha.edu | Prototype New CHAOSS Metrics | [GeorgLink/governance](https://github.com/GeorgLink/governance) | [no document](https://github.com/chaoss/governance/blob/master/GSoC-Ideas.md) |
+
```

Comment:
These three lines are breaking the table. Please fix (and squash, and maybe rebase, the commits) so that we can merge...

(Thanks for your interest!)